### PR TITLE
Joke timer

### LINKED ISSUE #11: [TASK] Modify the Joke of the Day so that it only updates once every 24hours
As a Site Owner,

I WANT the landing page Joke of the Day to update just once a day.
SO THAT I limit the API requests to once per day, and do use to many API calls.

**AC**
GIVEN I visit the Recipe Champion landing page multiple times.
WHEN I am on the landing page
THEN an API call for the Joke of the Day is only made 1 once every 24 hours.

### PR DESCRIPTION
**Pull request by ghost**
1. Modified joke call -
         Converted joke call to a conditional based on a stored time. 
         Stores the joke for access on index.html load unless time is greater than 24hrs. 
         Added some alerts and a minor style change for width of search bar. Something isn't working, not sure if I have the current 
         main... testing.
         Added the whole left column container div (#contentContainer) as that was lost somewhere.
2. Fixed -
          data storage after moving localStorage line, function call, and added history-container to main.html, I didn't have it for 
          some reason.

p.s. Just noticed the 10% gap on the right as 60+30=90, but this is a styling issue, I'll hold off on changing.

### CODE REVIEW

File: assets/js/javascript.js [hunk 1]
```diff
@@ -136,12 +144,14 @@ function apiCall(baseUrl, params = {}) {
   return fetch(requestUrl).then( function(response) {
     if (!response.status == 200) {
       // TODO: 404 Redirect
+      alert('Spoonacular responded with status 200');
```
Comment (Myuze) [inline assets/js/javascript.js]: This alert should display response.status as a variable, this only displays if the response.status was NOT 200 (not successful).
```suggestion
      alert(`Spoonacular responded with ${response.status}`);
```

File: assets/js/javascript.js [hunk 2]
```diff
@@ -151,10 +161,9 @@ function apiCall(baseUrl, params = {}) {
 
 function processSpoonacularData(data) {
   // TODO: Expand to handle various API calls
-  localStorage.setItem('queryArray', JSON.stringify(data));
   recipeCardBuild(data);
   console.log(data)
-  return data;
+  //return data; //<--don't think this is necessary, unless the call needs it for something
```
Comment (Myuze) [inline assets/js/javascript.js:166]: Yes, that's fine, just remove it.

File: assets/js/javascript.js [hunk 3]
```diff
@@ -197,38 +204,49 @@ $('#rebuildCards').click( function rebuildCards () {
...
 findRecipeBtn && findRecipeBtn.on('click', goToMain)
 
-// Add JOTD to landing page
+// Add JOTD to landing page, check if 24hours has passed since last call
 if (window.location.pathname.endsWith('index.html')) {
-  addJoke();
+  if (localStorage.getItem('day1') == null) { 
+    localStorage.setItem('day1', Date.now());
+    addJoke();
+  } else if (localStorage.getItem('day1') != null) {
```
Comment (Myuze) [inline assets/js/javascript.js]:
```suggestion
  } else {
```
This second If statement seems redundant. Ether something exists in localstorage or it's null.

File: assets/main.html [hunk 4]
```diff
@@ -39,8 +39,9 @@
                 <p class="lead text-center">Enter ingredients and search for new recipes</p>
                 <input id="main-search-input" type="text" placeholder="milk, eggs, butter, cinnamon">
                 <button id="main-search-btn" class="button expanded">Find recipes</button>
-            <!--wrapper for just the search history-->
             </div>
+            <!--wrapper for just the search history-->
+            <div class=".history-container"></div>
```
Comment (Myuze) [inline assets/main.html]: Typo? Erroneous "."
```suggestion
            <div class="history-container"></div>
```

File: assets/css/style.css [hunk 5]
```diff
@@ -31,7 +31,7 @@
     display: flex;
     flex-direction: column;
     justify-content: flex-start;
-    width: 30%;
```
Comment (mrogers23458) [inline assets/css/style.css:34]: I actually fixed this issue on my end as well. width display isn't necessary. There are fixed widths on elements that will set the size of the box.

File: assets/css/style.css [hunk 6]
```diff
@@ -92,7 +92,7 @@ with what spoontacular gives us. Selectors for applying styles are important */
 #contentContainer {
     display: flex;
     flex-direction: column;
-    width: 100%;
```
Comment (mrogers23458) [inline assets/css/style.css:95]: same as above

File: assets/main.html [hunk 7]
```diff
@@ -39,20 +39,24 @@
...
         </div>
-        <!--Foundation CSS-->
-        <div class="work-feature-block row">
-            <h2 class="work-feature-block-header text-center">You're just one click away from the perfect recipe!</h2>
-            <div class="columns medium-7">
-                <div class="word-cloud-center">
-                    <!--sample image placeholder that will hopefully be replaced by dynamically generated word cloud-->
-                    <!--dynamically generated content should be given this class, or appended here-->
-                    <img class="work-feature-block-image" src="./sampleWordCloudImage.jpg"/>
+        <!--Container for right side-->
```
Comment (mrogers23458) [inline assets/main.html:46]: Everything was displaying fine on my end, why are we adding another div here?

**Review by mrogers23458 — APPROVED** [review]

**Review by Myuze — APPROVED** [review]
Fixed Changes to move ahead.

**Pull request merged**